fk/luae_na/src/main/java/se/inera/intyg/common/luae_na/v1/model/converter/certificate/question/QuestionGrundForMUUnderlag.java

```diff
@@ -0,0 +1,173 @@
+                            .label(textProvider.get(GRUNDFORMU_JOURNALUPPGIFTER_LABEL))
+                            .build(),
+                        CheckboxMultipleDate.builder()
+                            .id(GRUNDFORMEDICINSKTUNDERLAG_ANHORIGS_BESKRIVNING_SVAR_JSON_ID_1)
+                            .label(textProvider.get(GRUNDFORMU_ANHORIG_BESKRIVNING_LABEL))
+                            .build(),
+                        CheckboxMultipleDate.builder()
+                            .id(GRUNDFORMEDICINSKTUNDERLAG_ANNAT_SVAR_JSON_ID_1)
+                            .label(textProvider.get(GRUNDFORMU_ANNAT_LABEL))
+                            .build()
+                    )
+                )
+                .build()
+            )
+            .value(
+                CertificateDataValueDateList.builder()
+                    .list(addValues(undersokningPatient, journaluppgifter, beskrivningPatient, annat))
+                    .build()
+            )
+            .validation(
+                new CertificateDataValidation[]{
+                    CertificateDataValidationMaxDate.builder()
+                        .id(GRUNDFORMEDICINSKTUNDERLAG_UNDERSOKNING_AV_PATIENT_SVAR_JSON_ID_1)
+                        .numberOfDays(NUMBER_OF_DAYS_IN_FUTURE)
+                        .build(),
+                    CertificateDataValidationMaxDate.builder()
+                        .id(GRUNDFORMEDICINSKTUNDERLAG_JOURNALUPPGIFTER_SVAR_JSON_ID_1)
+                        .numberOfDays(NUMBER_OF_DAYS_IN_FUTURE)
+                        .build(),
+                    CertificateDataValidationMaxDate.builder()
+                        .id(GRUNDFORMEDICINSKTUNDERLAG_ANHORIGS_BESKRIVNING_SVAR_JSON_ID_1)
+                        .numberOfDays(NUMBER_OF_DAYS_IN_FUTURE)
+                        .build(),
+                    CertificateDataValidationMaxDate.builder()
+                        .id(GRUNDFORMEDICINSKTUNDERLAG_ANNAT_SVAR_JSON_ID_1)
+                        .numberOfDays(NUMBER_OF_DAYS_IN_FUTURE)
+                        .build(),
+                    CertificateDataValidationMandatory.builder()
+                        .questionId(GRUNDFORMEDICINSKTUNDERLAG_TYP_DELSVAR_ID_1)
+                        .expression(multipleOrExpression(
+                            GRUNDFORMEDICINSKTUNDERLAG_UNDERSOKNING_AV_PATIENT_SVAR_JSON_ID_1,
+                            GRUNDFORMEDICINSKTUNDERLAG_JOURNALUPPGIFTER_SVAR_JSON_ID_1,
+                            GRUNDFORMEDICINSKTUNDERLAG_ANHORIGS_BESKRIVNING_SVAR_JSON_ID_1,
+                            GRUNDFORMEDICINSKTUNDERLAG_ANNAT_SVAR_JSON_ID_1))
+                        .build()
+                }
+            )
+            .build();
+    }
+
+    private static boolean validDate(InternalDate date) {
+        return date != null && date.isValidDate();
+    }
+
+    private static List<CertificateDataValueDate> addValues(InternalDate undersokningPatient, InternalDate journaluppgifter,
```

Comment:
Misleading method name.. it doesn't add anything but instead 'gets'. Consider renaming.